addons/mail/static/src/models/composer/composer.js

```diff
@@ -108,17 +96,8 @@ function factory(dependencies) {
                     thread.id === res_id &&
                     thread.model === res_model
                 );
-                return otherThread.composer.postMessage(Object.assign({}, data));
+                return otherThread.composer.postMessage();
```

Comment:
This line is not going to work because it would be expected to post the data of current composer but in the other thread (and not the data from the other composer), but I think this whole `if (thread.model === 'mail.box') {` is dead code that should be removed.

I don't think we ever display a composer with a fake thread (to confirm), and if we do, it should be forbidden.

Reply:
That was my concern but @alexkuhn was saying it's ok https://github.com/odoo/odoo/pull/53452#discussion_r444293375

Reply:
I said it was safe to remove `data` and `options` in this comment, but it's also safe to remove the `if (thread.model === 'mail.box')` altogether. Old code had discuss inbox composer with inbox as the thread, but now the composer is correctly linked to origin thread of message in inbox, so it's deadcode.